### PR TITLE
Feature/forest integrity accordion tabs

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: src/landscape/Forest.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  const componentsArray = [
+    {
+      label: {
+        id: 'forestLossPersistence',
+        name: 'Perdida y persistencia',
+        disabled: false,
+        expandIcon: <AddIcon />,
+        detailId: 'Perdida y persistencia',
+        description: 'Perdida y persistencia',
```
Comment (erikasv) [inline src/landscape/Forest.jsx:23]: This 3 properties aren't used in LandscapeAccordion component.
It's better to remove them from all 3 components (LandscapeAccordion, Forest and HumanFootprint)

File: src/landscape/Landscape.jsx [hunk 2]
```diff
@@ -92,6 +93,21 @@ class Landscape extends React.Component {
...
         ),
       },
+      {
+        label: {
+          id: 'forest',
+          name: 'Bosques',
+          disabled: false,
+          expandIcon: <AddIcon />,
+          detailId: 'Bosques',
+          description: 'Bosques',
```
Comment (erikasv) [inline src/landscape/Landscape.jsx:103]: Remove them also from this component

File: src/landscape/forest/ForestIntegrity.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  };
+
+  render() {
+    const { showInfoGraph } = this.state;
+    return (
+      <div className="graphcontainer pt6">
+        <h2>
+          <InfoIcon
+            className="graphinfo"
+            data-tooltip
```
Comment (erikasv) [inline src/landscape/forest/ForestIntegrity.jsx:35]: remove useless attribute

File: src/landscape/forest/ForestIntegrity.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            title="¿Qué significa este gráfico?"
```
Comment (erikasv) [inline src/landscape/forest/ForestIntegrity.jsx:36]: title attribute doesn't work on material ui icons, you need to wrap it with tooltip and set the title on it.

This is already implemented in release branch, you should merge release into develop (maybe create another PR after HF texts are updated, but only the PR into develop) and rebase his branch

File: src/landscape/forest/ForestIntegrity.jsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+          />
+        </h2>
+        {(
+          showInfoGraph && (
+          <ShortInfo
+            name="Integridad"
+            description="Integridad"
+            className="graphinfo2"
+            tooltip="¿Qué significa?"
+            customButton
```
Comment (erikasv) [inline src/landscape/forest/ForestIntegrity.jsx:54]: This prop changed in release branch

File: src/landscape/forest/ForestIntegrity.jsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+            title="¿Qué significa este gráfico?"
+            onClick={() => this.toggleInfoGraph()}
+          />
+          <div
+            className="graphinfo"
+            onClick={() => this.toggleInfoGraph()}
+            onKeyPress={() => this.toggleInfoGraph()}
+            role="button"
+            tabIndex="0"
+          />
```
Comment (erikasv) [inline src/landscape/forest/ForestIntegrity.jsx:45]: What is this div for?

File: src/landscape/forest/ForestLossPersistence.jsx [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+  };
+
+  render() {
+    const { showInfoGraph } = this.state;
+    return (
+      <div className="graphcontainer pt6">
+        <h2>
+          <InfoIcon
+            className="graphinfo"
+            data-tooltip
```
Comment (erikasv) [inline src/landscape/forest/ForestLossPersistence.jsx:35]: Same comments in ForestIntegrity

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/landscape/forest/ForestIntegrity.jsx (hunk 6 above)
Comment (cazapatamar) [reply src/landscape/forest/ForestIntegrity.jsx:45]: Don,t know, i'm removing it not only from forest new components but also the others

**Comment by cazapatamar** [comment]
Will be worked on other PR

**Pull request closed without merging**